chapters/api_concepts.txt

```diff
@@ -312,16 +318,15 @@ The following infos of objects can be queried:
 | description   | `STRING`         |    No    | explanation of the object, e.g., for a tooltip
 | parameter     | `PARAMETER_LIST` |    Yes   | list of supported parameters (array of `ANARIParameter`)
 | channel       | `STRING_LIST`    | for ``FRAME`` | list of supported channels
-| feature       | `STRING_LIST`    | for ``DEVICE``s and ``RENDERER``s | list of features supported by this object
+| feature       | `STRING_LIST`    | for ``RENDERER``s | list of features supported by this object
```

Comment:
list of supported features

Reply:
Changed in eaac0f744a138d5635c3636e59e8d1c2cb4e12a0